src/views/Onboarding/CreateMesh.vue

```diff
@@ -10,7 +10,7 @@
 
       <div class="flex justify-center mt-10 mb-12 pb-12">
         <div class="w-full sm:w-3/5 lg:w-2/5 p-4">
-          <KTable
+          <KTableLegacy
```

Comment:
Do we have an issue to update this? Why is this legacy?

Reply:
We do not, yet. I don’t think this is a pressing issue so far. It was easier to migrate using the legacy table because using the new KTable component requires some setup (see https://kongponents.konghq.com/components/table.html) and quite some changes. I’d rather first update to Vue 3 before considering using it.